test/test_force.js

```diff
@@ -46,35 +47,34 @@ function main(args) {
     var testProduction = parsedArgs.hasOwnProperty('test-production');
     var useSfdxRequested = parsedArgs.hasOwnProperty('use-sfdx');
     var chosenOperatingSystems = cleanSplit(parsedArgs.os, ',').map(function(s) { return s.toLowerCase(); });
-    var appTypes = parsedArgs.apptype || '';
     var templateRepoUri = parsedArgs.templaterepouri || '';
     var pluginRepoUri = !testProduction ? (parsedArgs.pluginrepouri || SDK.tools.cordova.pluginRepoUri) : '';
     var sdkBranch = parsedArgs.sdkbranch || defaultSdkBranch;
     var chosenAppTypes = cleanSplit(parsedArgs.apptype, ',');
+    var chosenClis = cleanSplit(parsedArgs.cli, ',');
 
-    var testingWithAppType = chosenAppTypes.length > 0 && templateRepoUri == '';
```

Comment:
The validation method prevents the user from entering an apptype and a templaterepouri so the check for templateRepoUri == '' is no longer needed.